ush/build.sh

```diff
@@ -12,7 +12,7 @@ COMPILER=${COMPILER:-"intel"}
 BUILD_DIR=${BUILD_DIR:-"${DIR_ROOT}/build"}
 INSTALL_PREFIX=${INSTALL_PREFIX:-"${DIR_ROOT}/install"}
 GSI_MODE=${GSI_MODE:-"Regional"}  # By default build Regional GSI (for regression testing)
-ENKF_MODE=${ENKF_MODE:-"GFS"}     # By default build Global EnKF  (for regression testing)
+ ENKF_MODE=${ENKF_MODE:-"GFS"}     # By default build Global EnKF  (for regression testing)
```

Comment:
I don't think this extra white space change is meant to be part of this PR.

Reply:
@RussTreadon-NOAA  please see the above explanation.
If a cleaner PR is needed, I will try to reverse it.

Reply:
Yes, please revert this change.  `ush/build.sh` should not be part of this PR.   

The simplest thing to do is to remove the white space from `ush/build.sh` in a local working copy of your branch, commit this change, and push the local copy of your branch to github.   Then we should not see `ush/build.sh` in the list of files changed.

Reply:
@RussTreadon-NOAA   Thanks for your suggestions. Seems some tricky issues occured and I found the easier to open another PR with a different branch.